Week-12/todo-list/src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
```

Comment:
we can remove this file and its uses, as its not required in our assignment.

Reply:
Removed